### DISCUSSION TITLE
No option found to remove my credit card after Copilot trial

### DISCUSSION THREAD

**Discussion by ghost** [community/community#32429] [Copilot Conversations]
I want to remove my credit card, I have no pending payment and don't want to continue using Copilot, asked four days ago to the support but didn't get any answer.
Why there is no evident way to remove a credit card from account?

If I go to "Billing and plans" > "Payment information" > "Payment method" there is just an "edit" link and only a form to replace the payment method. Not delete it...

**Comment by iamcjwilliams** [comment]
Hello @ed-db We're sorry for the trouble here. Our copilot team is currently experiencing a high volume of tickets, so responses will be delayed. We will respond as soon as possible. Thank you for your patience
